### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ShivamAmratlalPatel**

### CODE REVIEW

File: frontend/src/components/Login.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<template>
+<div>
+  <button type="button" @click="Login">Login {{ message }}</button>
```
Comment (ShivamAmratlalPatel) [inline frontend/src/components/Login.vue:3]: Can remove {{ message }}. That was to add a custom message

Comment (ShivamAmratlalPatel) [inline frontend/src/components/Login.vue:3]: Inside the button tag you need to add the v-if

Comment (ShivamAmratlalPatel) [inline frontend/src/components/Login.vue:3]: Also the method on @click right now is set to Login however the method below is called ChangeLogin

File: frontend/src/components/Login.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<script>
+export default {
+    data() {
+        return {
+            isLoggedIn: false
+        }
+    },
+    methods: {
+        changeLogin() {
+            return false
```
Comment (ShivamAmratlalPatel) [inline frontend/src/components/Login.vue:18]: The logic here needs to be written. Consider what changeLogin needs to do depending on what isLoggedIn value shows

File: frontend/src/components/Login.vue (hunk 1 above)
Comment (ShivamAmratlalPatel) [reply frontend/src/components/Login.vue:3]: You can use the value isLoggedIn within the v-if

File: frontend/src/style.css [hunk 3]
```diff
@@ -40,11 +40,6 @@ body {
   min-height: 100vh;
 }
 
-h1 {
```
Comment (ShivamAmratlalPatel) [inline frontend/src/style.css:43]: Why did you remove this css styling

File: frontend/src/style.css [hunk 4]
```diff
@@ -55,6 +50,9 @@ button {
   background-color: #1a1a1a;
   cursor: pointer;
   transition: border-color 0.25s;
+  position: absolute;
```
Comment (ShivamAmratlalPatel) [inline frontend/src/style.css:53]: so this will make every button follow this css styling. However we probably only want to set this styling for the login button. This can be done with the style parameter inside the button tag

File: frontend/src/style.css (hunk 3 above)
Comment (dhyeyd1) [reply frontend/src/style.css:43]: I think it might have been by accident as i was tampering with code to try and place the login button in towards the top right corner
